webclient/src/forms/RequestPasswordResetForm/RequestPasswordResetForm.tsx

```diff
@@ -26,9 +26,13 @@ const RequestPasswordResetForm = ({ onSubmit, skipTokenRequest }) => {
     setIsMFA(true);
   }, ServerTypes.RESET_PASSWORD_CHALLENGE, []);
 
-  const handleOnSubmit = (form) => {
+  const handleOnSubmit = ({ userName, email, ...values }) => {
     setErrorMessage(false);
-    onSubmit(form);
+
+    userName = userName.trim();
+    email = email.trim();
```

Comment:
Similarly to `number` above, could this field be set to `type="email` to avoid an explicit trim?

Reply:
This one is missing the `type="email"` attribute, i can add it. Similarly, do you think this protects against the need for using trim?

@ZeldaZach any issue with adding the type=email attr? This will enforce the browsers email validation.

Reply:
why wouldn't we? I see no problem with that

Reply:
We were both wrong about this, `number` does protect us from the need for trimming, but something with React is causing type email to not be validated, nor does it protect us from needing to trim.